test/integration/select_line_test.rb

```diff
@@ -24,6 +24,21 @@ class SelectLineTest < ActionDispatch::IntegrationTest
     end
   end
 
+  # TODO
+  describe 'line redirect on single line' do
+    before do
+      # TODO setup line array like so ['dc']
+      @user.line='DC'
+      # LINES=%w(DC).map(&:to_sym).freeze
+      puts @user.inspect
+      # how to change env var constants in this block?
```

Comment:
hmm, this is tricky! `LINES` gets pulled from an env var `LINES` and gets set on app boot/in an initializer iirc, so I'd bet what we're looking to do here is, within the scope of this test case, set `LINES` to just 'DC' like you're doing in the commented out thing. 

I don't think rails boots up and down for every test, so I would bet that doing `ENV['LINES'] = [:DC].freeze` won't work. but worth a shot! If it's not a free basket I'll poke around a little.